src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,214 @@
+      newProps.onClose();
+    }
+  }
+
+  componentDidMount() {
+    this.setAccessibilityAttributes();
+    addEventListener(document, 'keyup', this.handleKeyEvent);
+  }
+
+  componentDidUpdate() {
+    this.setAccessibilityAttributes();
+  }
+
+  componentWillUnmount() {
+    removeEventListener(document, 'keyup', this.handleKeyEvent);
+  }
+
+  render() {
+    const { 
+      activatorWrapper: WRAPPERCOMPONENT = 'div',
+      active,
+      children,
+      DropdownItems,
+      toggle,
+      direction,
+      disabled
+    } = this.props;
+    
+    const {
+      selectedIndex
+    } = this.state;
+
+    
+    const dropdownClassName = classNames (
+      typeof direction == 'undefined' || direction === "down" ? baseTheme.dropdown 
+        : direction === "up" ? baseTheme.dropup
+        : direction === "left" ? baseTheme.dropleft
+        : baseTheme.dropright,
+       !disabled && active && baseTheme.active
+    );
+
+    const dropdownMenuClassName = classNames (
+      baseTheme.dropdownMenu,
+      !disabled && active && baseTheme.active,
+    );
+
+    const DropdownItemComponents = DropdownItems.map((item,index) => 
+            <DropdownItem 
+              key={index}
+              active={selectedIndex === index} 
+              disabled={item.disabled}
+              divider={item.divider}
+              header={item.header}
+              children={item.children}
+            ></DropdownItem>
+        );
+      
+    return (
+      <WRAPPERCOMPONENT ref={this.setActivator}>
+        <div className={dropdownClassName} key={this.id}>
+          <Button onClick={toggle}>{children}</Button>
+          <div className={dropdownMenuClassName}>
+            {DropdownItemComponents}
+          </div>
+        </div>
+      </WRAPPERCOMPONENT>
+    );
+  }
+
+  private setAccessibilityAttributes() {
+    const { id, activatorContainer } = this;
+    if (activatorContainer == null) { return; }
+
+    const firstFocusable = findFirstFocusableNode(activatorContainer);
+    const focusableActivator = firstFocusable || activatorContainer;
+
+    focusableActivator.tabIndex = 0;
+    focusableActivator.setAttribute('aria-controls', id);
+    focusableActivator.setAttribute('aria-owns', id);
+    focusableActivator.setAttribute('aria-haspopup', 'true');
+    focusableActivator.setAttribute('aria-expanded', String(this.props.active));
+  }
+
+  @autobind
+  private setActivator(node: HTMLElement | null) {
+    if (node == null) {
+      this.activatorContainer = null;
+      return;
+    }
+    this.activatorContainer = node;
+  }
+
+  @autobind
+  private handleKeyEvent(event: KeyboardEvent) {
+    const {
+      active,
+      toggle,
+      disabled
+    } = this.props;
+
+    const {
+      selectedIndex
+    } = this.state;
+
+    if (!active && !disabled) {
+      return;
+    }
+    
+    // Change Items on key up, down and tab
+    if (event.keyCode === Keys.UP_ARROW) {
```

Comment:
@shyamk9 When I press up & down arrow keys then it also move the whole body up & down that should be disabled

Reply:
ok, but i don't know how to do that.

Reply:
@shyamk9 you can check in react-strap, go to there github & take a look on how they have handled it, basically here you are not looking for an event for any specific element, it seems you are checking that in document level, that could be the one reason, but still check there repo